include/reactor-uc/reactor-c.h

```diff
@@ -0,0 +1,36 @@
+#ifndef REACTOR_C_H
+#define REACTOR_C_H
+
+#define lf_print(str, ...) printf(str, __VA_ARGS__)
+#define PRINTF_TIME "%" PRId64
+#define PRINTF_TAG "(" PRINTF_TIME ", " PRINTF_MICROSTEP ")"
```

Comment:
Dont we need to define PRINTF_MICROSTEP also? Should by PRId32 I think

Reply:
you are right - fixed